### ISSUE TITLE
synchronous backend?

### ISSUE THREAD

**Issue by c-cube** [mirage/capnp-rpc#188]
some projects don't use lwt (lots of synchronous code). A blocking backend would be very useful.

**Comment by talex5** [comment]
The core `capnp-rpc` library doesn't depend on Lwt, but you'd still need to replace it with something else. Fundamentally, capnp-rpc is a promise library (like Lwt), but in which the promises can also be remote. For blocking programs, you'd probably be better off using plain capnp (without the RPC stuff).

**Comment by c-cube** [comment]
Fair enough, my use case is clients that don't need concurrency (open one connection and pretend calls are blocking).

**Comment by LasseBlaauwbroek** [comment]
I would also be interested in a blocking (non-lwt) backend.
